minifi/KEYS

```diff
@@ -0,0 +1,404 @@
+This file contains the PGP keys of various developers.
```

Comment:
this file should go away completely.  The KEYS file of the NiFi dir itself is sufficient.